modules/layers/src/column-layer/column-geometry.js

```diff
@@ -66,8 +66,18 @@ function tesselateColumn(props) {
         i += 3;
       }
     }
+
+    // duplicate the last vertex to create proper degenerate triangle.
+    positions[i + 0] = positions[i - 3];
```

Comment:
Why is this needed? The last two vertices were bottom v0 and top v0, and the next vertex will be top v0.

Reply:
> What do we need to put in between, in order to link up the triangles? We’ll need an even number of new triangles in order to preserve the winding. We can do this by repeating the last vertex of the first row, and the first vertex of the second row

https://www.learnopengles.com/tag/degenerate-triangles/